tests/queries/test_db_returning.py

```diff
@@ -49,3 +49,12 @@ def test_bulk_insert(self):
             with self.subTest(obj=obj):
                 self.assertTrue(obj.pk)
                 self.assertIsInstance(obj.created, datetime.datetime)
+
+
+@skipUnlessDBFeature('can_return_columns_from_insert')
+class DatabaseDefaultsTests(TestCase):
+    def test_now(self):
+        obj = ReturningModel.objects.create()
+        self.assertIsInstance(obj.created, datetime.datetime)
+        self.assertIsInstance(obj.year, int)
+        self.assertEquals(obj.pi, 3.14159265358979)
```

Comment:
[`assertEquals` is a deprecated alias](https://docs.python.org/3/library/unittest.html?highlight=assertequals#deprecated-aliases)


```suggestion
        self.assertEqual(obj.pi, 3.14159265358979)
```